### DISCUSSION TITLE
Mi mouse no se mueve bien

### DISCUSSION THREAD

**Discussion by olimar232** [ppy/osu#25389] [Q&A]
Mi mouse cuando juego al osu tiene momentos en el que no se mueve y despues sigue como si nada. Por ejemplo: yo al hacer circulos con el mouse en el escritorio funciona bien. Pero al hacerlo en el osu los circulos que hago con el mouse se van moviendo para arriba / abajo / los costados.

**Comment by bdach** [comment]
en inglés por favor

please attach video of this with frame graph shown (ctrl-f11 twice)
